packages/xstate-wallet/src/ui/approve-budget-and-fund-workflow.tsx

```diff
@@ -71,11 +71,14 @@ export const ApproveBudgetAndFund = (props: Props) => {
         </Box>
       </Flex>
       <Text fontSize={1} pb={2}>
-        The app will have full control to manage these funds on your behalf.
+        <strong>{budget.domain}</strong> will manage these funds.
       </Text>
       <Text pb={3} fontSize={1}>
-        You will need to deposit {formatEther(playerAmount)} ETH into a channel with a state channel
-        hub.
+        You will deposit {formatEther(playerAmount)} ETH into a channel. Our hub will also make a
+        deposit.
+        <Tooltip message="This allows you to transact with anyone else connected to the same hub.">
+          <Icon name="Info" size="20" />
+        </Tooltip>
```

Comment:
This seems like the right moment to introduce the hub. But there is limited real estate on this modal. Therefore, trim the wording down a bit and put some additional information in a tooltip wrapping a [ ? ] icon. 
![Screenshot 2020-06-18 at 15 02 17](https://user-images.githubusercontent.com/1833419/85030045-e2589400-b174-11ea-88cb-76a45b368b05.png)
![Screenshot 2020-06-18 at 15 02 23](https://user-images.githubusercontent.com/1833419/85030050-e4225780-b174-11ea-907a-11a201abab3b.png)